app/models.py

```diff
@@ -53,13 +54,15 @@ def __repr__(self):
         return '<{}: {}>'.format(self.__class__.__name__, self.name)
 
     def serialize(self):
-        return {
+        lot = {
             'id': self.id,
             'slug': self.slug,
             'name': self.name,
             'oneServiceLimit': self.one_service_limit,
             'allowsBrief': self.allows_brief,
         }
+        lot.update(dict(self.data.items()))
```

Comment:
Should we follow the same rules we use for services: start with `data = dict(self.data.items())` and then update with the column fields dictionary above, so that column values always override the JSON data if there's a collision?